src/containers/my-resources/attachments-container/AttachmentsContainer.tsx

```diff
@@ -89,10 +85,7 @@ const AttachmentsContainer = () => {
       data={{ items: response.items }}
       emptyTableKey='myResourcesPage.emptyAttachments'
       rowActions={rowActions}
-      sort={{
-        sort,
-        onRequestSort
-      }}
+      sort={{ sort, onRequestSort }}
```

Comment:
As i understand you can pass just `sortOptions` here:
```tsx
sort={sortOptions}
```
and that would be enough)